config/samples/sw.kogito_v1alpha08_kogitoserverlessplatformWithImageSnapshotOverride.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: sw.kogito.kie.org/v1alpha08
+kind: KogitoServerlessPlatform
```

Comment:
these three examples are basically the same, maybe we can have just one?

Reply:
ok